### ISSUE TITLE
Mitglieder portal "wo ist Arbeit möglich?"

### ISSUE THREAD

**Issue by PascaleGG** [OpenOlitor/OpenOlitor#149]
**Auf dem Mitgliederportal: "Wo ist Arbeit möglich? - vergangenen und kommenden Daten sind angezeigt, das macht das ganze unübersichtlich**

**Lösung: vergangenen Daten ausblenden/hide automatically**

**Mögliche Alternativen / Possible Alternatives**
Ich  habe bisher die Arbeitsangebote zurückgezogen, jeweils, wenn das Datum vorbei war. Das, ein Angebot nach dem anderen, einezln, weil das nicht "gruppiert ging".  -> viel zu aufwendig, macht kein Sinn. 
Die Arbeitsangeboten wollen wir behalten (= löschen ist keine Alternative).

Could you please make the Arbeitsangebot dissapear on the Kundenportal, once the date has passed? 
Thank you for your help.

**Comment by alexander-nbg** [comment]
It becomes likewise cluttered on our instance. However, people will still need to see their support days roughly 1 year into the past to check with what they should support.
An adjustable filter "show only  dates that are up to X months in the past" might be a solution.

**Comment by mcmpp** [comment]
Hi @alexander-rd and @PascaleGG ,

I have made a proposal and deployed it in test: try.openolitor.ch. 
Could you have a look and let me know if this change covers your need??

**Comment by alexander-nbg** [comment]
I was just about cleaning up my mailbox and this one got lost. 

I see only "Arbeitsangebote" in the future so this seems to fit the request from @PascaleGG but I cannot see if there is some option to see the one year into the past. I would still like to have members seeing about 1 year into the past.

Since I see a switch "ausstehend/alle" for "Abwesenheiten" above, this switch may also work well for the "1 year into the past" request. It would fulfill the needs.

**Comment by mcmpp** [comment]
Hi @alexander-rd , 

can you check now if the current version fits your needs?

**Comment by alexander-nbg** [comment]
You've added "Geschäftsjahre" in the section "Wo ist Mitarbeit möglich?"? 

The option, in general, should work well but should be applied to the section "Deine Arbeitseinsätze". :D

**Comment by mcmpp** [comment]
@alexander-rd  I have added this filter, can you please check it

**Comment by alexander-nbg** [comment]
Yes, this should work with the "Geschäftsjahr" fitler on the "Deine Arbeitseinsätze" view.
![grafik](https://github.com/OpenOlitor/OpenOlitor/assets/525857/13775aaa-4224-4d22-ac1b-31bd192058f2)

Where is the time frame for the "Geschäftsjahr" set? I remember that our's may not be set to the calendar year. This is something that's not very transparent to our crop sharers and something, I should align before changes take effect.
